src/app/clusters/administrator-commissioning-server/administrator-commissioning-server.cpp

```diff
@@ -46,36 +46,41 @@ bool emberAfAdministratorCommissioningClusterOpenCommissioningWindowCallback(
     auto & salt                 = commandData.salt;
     auto & passcodeID           = commandData.passcodeID;
 
-    EmberAfStatus status = EMBER_ZCL_STATUS_SUCCESS;
+    Optional<StatusCode> status = Optional<StatusCode>::Missing();
```

Comment:
Fwiw, Missing is the default-constructed value for Optional, so could just leave it off...

Reply:
One of the example apps build was failing unless I add the Missing here. It complained about uninitialized `status` value (for example, when there is no error).

Reply:
That is ... so bizarre.  `Optional` has a default constructor!